src/OrchardCore.Modules/OrchardCore.Lists/AdminNodes/ListsAdminNodeNavigationBuilder.cs

```diff
@@ -94,7 +94,7 @@ private async Task AddContentItemsAsync(NavigationBuilder listTypeMenu)
             {
                 var cim = await _contentManager.PopulateAspectAsync<ContentItemMetadata>(ci);
 
-                if (cim.AdminRouteValues.Any() && ci.DisplayText != null)
+                if (cim.AdminRouteValues.Count > 0 && ci.DisplayText != null)
```

Comment:
Why?

Reply:
When you call any, it would have to call the enumerator, the we are dealing with Array or List, we should use the Count it Length since they don't need to call Enumerator to get the result.

Reply:
hmm, ok. Well, maybe we took a bad habit of using .Any() everywhere. Basicallly we should use .Any for `IEnumerator<T>` and .Count for everything that inherits from ICollection.

Reply:
I'm surprised that it is still relevant in 2024 with .NET 8.

Reply:
Yes.  For arrays, use Length, for collection using Count. Any() should only be used with IEnumerable or when the other 2 do not exists

Reply:
.Any() calls Enumerable.Any(collection) behind the sense. 

Minor performance benefit but still in the framework I'll take any performance gain I can get.

Reply:
hmm, yeah, well I would optimize where it counts because ... sometimes it's just not worth it.

Reply:
Well yeah that's the plan